### PR TITLE
python3 script to add jobId and timestamp to redis

### PR DESCRIPTION
**Pull request by shabd67**
# Resolves

[CERES-651](https://rackspace.atlassian.net/browse/CERES-651)

# What

Automate initialization of redis downsampling jobs

# How

Added python3 script to add jobId and timestamp to redis which will be running at every new environment initialisation time.

### CODE REVIEW

**Review by rohitsngh27 — APPROVED** [review]
LGTM. Let's wait for @matsboman review.

**Comment by matsboman** [comment]
LGTM, what about the automation part? How and when will the python script be run?

**Comment by shabd67** [comment]
> LGTM, what about the automation part? How and when will the python script be run?



> LGTM, what about the automation part? How and when will the python script be run?

it will be running after Redis pods are up for a new environment.

File: redis-script/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+ENV REDISPORT=6379
+ENV REDISPASSWORD=""
+
+RUN mkdir /app
+ADD . /app
+WORKDIR /app
+
+RUN pip install --no-cache-dir -r installables.txt
+
+ENTRYPOINT python3 addTimeStampSetsToRedis.py $REDISHOST $REDISPORT $REDISPASSWORD
```
Comment (matsboman) [inline redis-script/Dockerfile:13]: missing line

**Review by matsboman — APPROVED** [review]

**Pull request merged**